src/Authorization/Services/Implementation/EventLogService.cs

```diff
@@ -53,7 +54,7 @@ public async Task CreateAuthorizationEvent(IFeatureManager featureManager, Xacml
             options.Converters.Add(new JsonStringEnumConverter());
             if (await featureManager.IsEnabledAsync(FeatureFlags.AuditLog))
             {
-                AuthorizationEvent authorizationEvent = EventLogHelper.MapAuthorizationEventFromContextRequest(contextRequest, context, contextResponse, _systemClock.UtcNow.DateTime);
+                AuthorizationEvent authorizationEvent = EventLogHelper.MapAuthorizationEventFromContextRequest(contextRequest, context, contextResponse, _timeProvider.GetUtcNow().ToOffset(TimeSpan.Zero));
```

Comment:
There is no point in doing `ToOffset` here, as `GetUtcNow` gives you a UTC datetimeoffset (offset = 0).